.gitignore

```diff
@@ -1,2 +1,7 @@
 *.test
-*.pprof
\ No newline at end of file
+*.pprof
+
+# Ignore seed corpus entries for these tests as we already have entries in tests
+# directly.
+testdata/fuzz/FuzzEncryptDecrypt
```

Comment:
No need to ignore testdata, keep the failures.

Reply:
Done. Thanks for looking at it!